WinUIGallery/ControlPages/XamlFundamentals/BindingPage.xaml

```diff
@@ -0,0 +1,394 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page x:Class="WinUIGallery.ControlPages.BindingPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:WinUIGallery.ControlPages"
+      xmlns:core="using:WinUIGallery"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <local:BooleanToVisibilityConverter x:Key="BoolToVisibility" />
+    </Page.Resources>
+
+    <StackPanel>
+        <Expander HorizontalAlignment="Stretch"
+                  HorizontalContentAlignment="Left"
+                  Margin="0 12 0 0">
+            <Expander.Header>
+                <TextBlock Text="Understanding binding in WinUI 3"
+                           Margin="0 24" />
+            </Expander.Header>
+            <StackPanel>
+                <RichTextBlock Margin="0 0 0 12">
+                    <Paragraph>
+                        <Bold>Key concepts:</Bold>
+                    </Paragraph>
+                    <Paragraph>
+                        <Run Text="1. " />
+                        <Bold>Target</Bold>
+                        <Run Text=": the property of a control that you want to bind to (e.g., " />
+                        <Italic>Text</Italic>
+                        <Run Text=", " />
+                        <Italic>Background</Italic>
+                        <Run Text=", " />
+                        <Italic>Visibility</Italic>
+                        <Run Text=")." />
+                    </Paragraph>
+                    <Paragraph>
+                        <Run Text="2. " />
+                        <Bold>Source</Bold>
+                        <Run Text=": the data you are binding to, like a property in a class, another control, or a static resource." />
+                    </Paragraph>
+                    <Paragraph>
+                        <Run Text="3. " />
+                        <Bold>Binding modes</Bold>
+                        <Run Text=":" />
+                    </Paragraph>
+                    <Paragraph>
+                        <Run Text="   - " />
+                        <Bold>OneWay</Bold>
```

Comment:
I think it's valuable to mention that the **default** modes for `Binding` and `x:Bind` are different (OneWay and OneTime, resp.) A developer converting Binding to x:Bind might have unwanted results.